packages/terra-tag/src/index.js

```diff
@@ -0,0 +1,5 @@
+import TagList from './TagList';
+import Tag from './Tag';
+
+export default Tag;
```

Comment:
Is this correct to make the default, Tag, when there are multiple components exported for this package?

Reply:
Before this PR, there was only one export, `Tag`. I suggested to make it the default export to maintain passivity.